pandas/core/nanops.py

```diff
@@ -1218,7 +1283,9 @@ def _zero_out_fperr(arg):
 
 
 @disallow("M8", "m8")
-def nancorr(a, b, method="pearson", min_periods=None):
+def nancorr(
+    a: np.ndarray, b: np.ndarray, method="pearson", min_periods: Optional[int] = None,
```

Comment:
method: str ?

Reply:
or Union[str, Callable]

Reply:
In https://github.com/pandas-dev/pandas/pull/30461/commits/7442310a426f9160abab5e201c10b8ad4590b25d I removed all annotations of ```str``` and ```Callable``` that were added in this PR.

No matter what annotation I try, mypy will complain, I will maybe be able to address this annotation in another PR, but I think that is out of scope for this PR at least.